blog-post.html

```diff
@@ -2,11 +2,92 @@
 <html>
   <head>
     <title>Blog Post</title>
+    <style>
+      @import "index.css";
+    </style>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
   </head>
   <body>
-    <section id="header"></section>
-    <section id="main"></section>
+
+    <section id="header">
+      <h1><a class="header-link" href="index.html">POST ONLY</a></h1>
+      <H6>A PAGE WITH JUST A BLOG POST CONTENT.</H6>
+    </section>
+<br>
+<br>
+    <section id="main4">
+      <div class="main-content4-wrapper">
+        <div class="main-content4">
+          <h4 class="main-content4-text1">THIS IS A <b>BLOG POST </b>ONLY LAYOUT</h4>
+        </div>
+        <div class="main-content4-img-container">
+          <img class="main-content4-img" src="https://picsum.photos/300/205"/>
+        </div>
+        <h5>AND YEAH, THAT'S BASICALLY IT</h5>
+        <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptatibus, reprehenderit vel natus molestiae non rem saepe. Quo deserunt laborum recusandae corporis, libero possimus, id voluptatem perferendis suscipit facere ipsam cumque accusamus numquam alias, aut natus iste corrupti sit. Reprehenderit, iusto mollitia ab inventore magnam totam dignissimos similique quasi placeat aut. Ullam at distinctio consequatur quos cum voluptatum itaque tenetur veniam odit, magnam maiores explicabo sint est, error natus dolor doloremque?</p>
+        <br>
+        <h5>MORE MORE MORE MORE</h5>
+        <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptas mollitia nobis vero corrupti voluptatum quia fugit, nisi nulla, dignissimos hic quisquam id!</p>
+        <p>Corporis dolore labore vero nobis iure obcaecati temporibus veritatis incidunt blanditiis, omnis praesentium velit, ab architecto natus veniam ipsam nostrum laudantium dolorem odio enim eligendi inventore atque impedit. Voluptas quo odio ipsum minima, a eos doloribus explicabo magnam voluptates nisi minus sint voluptatem eveniet reprehenderit doloremque pariatur commodi dolore autem asperiores. Placeat, odio!</p>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>

```

Comment:
This spacing could have been done with CSS and in a way that would be more reactive based on the size of the screen